app/views/devise/registrations/edit.html.erb

```diff
@@ -7,33 +7,43 @@
     <%= f.label :email %><br />
     <%= f.email_field :email, autofocus: true %>
   </div>
+  <br>
 
   <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
     <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+    <br>
   <% end %>
 
   <div class="field">
     <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
     <%= f.password_field :password, autocomplete: "off" %>
   </div>
+  <br>
 
   <div class="field">
     <%= f.label :password_confirmation %><br />
     <%= f.password_field :password_confirmation, autocomplete: "off" %>
   </div>
+  <br>
 
   <div class="field">
     <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
     <%= f.password_field :current_password, autocomplete: "off" %>
   </div>
+  <br>
 
   <div class="actions">
-    <%= f.submit "Update" %>
+    <%= f.submit "Update", class: "btn btn-lg btn-primary" %>
   </div>
 <% end %>
+<br>
```

Comment:
I talked to Fuzzy about form style. He says he is working on a standard form style for the app (on another feature branch). We agreed that we should merge this form as-is and once the standard style is done, go back and refactor this one. So when this PR is merged, can you please mark the issue as 'on hold' in redmine and add a note to this effect. Thanks!